pkg/controller/bastion/actuator.go

```diff
@@ -218,18 +218,18 @@ func deleteRule(openstackClientFactory openstackclient.Factory, ruleID string) e
 	return client.DeleteRule(ruleID)
 }
 
-func bastionConfigCheck(bastionConfig *controllerconfig.BastionComputeConfig) error {
-	config := controllerconfig.BastionComputeConfig{}
+func bastionConfigCheck(bastionConfig *controllerconfig.BastionConfig) error {
+	config := controllerconfig.BastionConfig{}
 	if *bastionConfig == config {
-		return errors.New("bastionConfig must not be empty in 00-componentconfig.yaml")
+		return errors.New("bastionConfig must not be empty")
 	}
 
 	if bastionConfig.FlavorRef == nil || *bastionConfig.FlavorRef == "" {
-		return errors.New("bastionComputeConfig.flavorRef must not be empty in 00-componentconfig.yaml")
+		return errors.New("bastionConfig.flavorRef must not be empty")
 	}
 
 	if bastionConfig.ImageRef == nil || *bastionConfig.ImageRef == "" {
-		return errors.New("bastionComputeConfig.flavorRef must not be empty in 00-componentconfig.yaml")
+		return errors.New("bastionConfig.flavorRef must not be empty")
```

Comment:
should be `bastionConfig.imageRef must not be empty` instead

Reply:
Thanks, committed in abcddfa9